test/routes/save-record.route.test.js

```diff
@@ -104,6 +110,25 @@ describe('/save-record route', () => {
 
         expect(response.headers.location).toEqual(nextUrl)
       })
+
+      it('should NOT save the record in the dataverse and redirect to the service complete page', async () => {
```

Comment:
Description should include the text 'when the payment has failed'

Reply:
fixed